modules/ichorcna/1.0/config/default.yaml

```diff
@@ -0,0 +1,120 @@
+                    grch37: "1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,X,Y"
+                    hs37d5: "1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,X,Y"
+                    hg38: "chr1,chr2,chr3,chr4,chr5,chr6,chr7,chr8,chr9,chr10,chr11,chr12,chr13,chr14,chr15,chr16,chr17,chr18,chr19,chr20,chr21,chr22,chrX,chrY"
+                    grch38: "chr1,chr2,chr3,chr4,chr5,chr6,chr7,chr8,chr9,chr10,chr11,chr12,chr13,chr14,chr15,chr16,chr17,chr18,chr19,chr20,chr21,chr22,chrX,chrY"
+                qual: 20
+                binSize:  1000000 # set window size to compute coverage 
+                # available binSizes are: 1000000, 500000, 50000, 10000
+            run:
+                ichorCNA_libdir: "{MODSDIR}/src/"
+                ichorCNA_rscript:  "{MODSDIR}/src/runIchorCNA.R"
+                # use panel matching same bin size (optional)
+                ichorCNA_normalPanel: 
+                    "1000000": "{MODSDIR}/src/inst/extdata/HD_ULP_PoN_{genome_build}_1Mb_median_normAutosome_median.rds"
+                    "500000": "{MODSDIR}/src/inst/extdata/HD_ULP_PoN_{genome_build}_500kb_median_normAutosome_median.rds"
+                # must use gc wig file corresponding to same binSize (required)
+                ichorCNA_gcWig:
+                    "1000000": "{MODSDIR}/src/inst/extdata/gc_{genome_build}_1000kb.wig"
+                    "500000": "{MODSDIR}/src/inst/extdata/gc_{genome_build}_500kb.wig"
+                    "50000": "{MODSDIR}/src/inst/extdata/gc_{genome_build}_50kb.wig"
+                    "10000": "{MODSDIR}/src/inst/extdata/gc_{genome_build}_10kb.wig"
+                # must use map wig file corresponding to same binSize (required)
+                ichorCNA_mapWig:  
+                    "1000000": "{MODSDIR}/src/inst/extdata/map_{genome_build}_1000kb.wig"
+                    "500000": "{MODSDIR}/src/inst/extdata/map_{genome_build}_500kb.wig"
+                    "50000": "{MODSDIR}/src/inst/extdata/map_{genome_build}_50kb.wig"
+                    "10000": "{MODSDIR}/src/inst/extdata/map_{genome_build}_10kb.wig"
+                # use bed file if sample has targeted regions, eg. exome data (optional)
+                ichorCNA_exons:  NULL
+                ichorCNA_centromere:  
+                    grch37: "{MODSDIR}/src/inst/extdata/GRCh37.p13_centromere_UCSC-gapTable.txt"
+                    hg19: "{MODSDIR}/src/inst/extdata/GRCh37.p13_centromere_UCSC-gapTable.txt"
+                    hs37d5: "{MODSDIR}/src/inst/extdata/GRCh37.p13_centromere_UCSC-gapTable.txt"
+                    grch38: "{MODSDIR}/src/inst/extdata/GRCh38.GCA_000001405.2_centromere_acen.txt"
+                    hg38: "{MODSDIR}/src/inst/extdata/GRCh38.GCA_000001405.2_centromere_acen.txt"
+                ichorCNA_minMapScore: 0.75
+                ichorCNA_chrs:  
+                    grch37: "c('1', '2', '3','4','5','6','7','8','9','10','11','12','13','14','15','16','17','18','19','20','21','22','X')"
```

Comment:
Just input-syntax for two different programs (the first one is formatted for readCounter as part of hmmcopy_utils and this format is for ichorCNA (to be used in R))

Reply:
Can this be moved to snakelike, instead of being in config? There, you can use file listing chromosomes generated by reference files (main_chromosomes.txt) or use function to generate chromosome names (there is example in sage module)

Reply:
Good suggestion. I agree it's better to embed anything that generates code (python, R etc) into the snakefile